lib/sandbox/postman-legacy-interface.js

```diff
@@ -20,34 +20,33 @@ const _ = require('lodash'),
     ],
 
     LEGACY_GLOBS_ALTERNATIVES = {
-        tests: 'pm.test()',
-        globals: 'pm.globals',
-        environment: 'pm.environment',
-        data: 'pm.iterationData',
-        request: 'pm.request',
-        responseCookies: 'pm.cookies',
-        responseHeaders: 'pm.response.headers',
-        responseTime: 'pm.response.responseTime',
-        responseCode: 'pm.response.code',
-        responseBody: 'pm.response.text()',
-        iteration: 'pm.info.iteration',
-        _: 'require(\'lodash\')',
-        CryptoJS: 'require(\'crypto-js\')',
-        tv4: 'require(\'ajv\')',
-        xml2Json: 'require(\'xml2js\')',
-        Backbone: 'require(\'backbone\')',
-        cheerio: 'require(\'cheerio\')',
-        'postman.setNextRequest': 'pm.execution.setNextRequest()',
-        'postman.setEnvironmentVariable': 'pm.environment.set()',
-        'postman.getEnvironmentVariable': 'pm.environment.get()',
-        'postman.clearEnvironmentVariables': 'pm.environment.clear()',
-        'postman.clearEnvironmentVariable': 'pm.environment.unset()',
-        'postman.setGlobalVariable': 'pm.globals.set()',
-        'postman.getGlobalVariable': 'pm.globals.get()',
-        'postman.clearGlobalVariables': 'pm.globals.clear()',
-        'postman.clearGlobalVariable': 'pm.globals.unset()',
-        'postman.getResponseCookie': 'pm.cookies.get()',
-        'postman.getResponseHeader': 'pm.response.headers.get()'
+        tests: '"pm.test()"',
+        globals: '"pm.globals"',
+        environment: '"pm.environment"',
+        data: '"pm.iterationData"',
+        request: '"pm.request"',
+        responseCookies: '"pm.cookies"',
+        responseHeaders: '"pm.response.headers"',
+        responseTime: '"pm.response.responseTime"',
+        responseCode: '"pm.response.code"',
+        responseBody: '"pm.response.text()"',
+        iteration: '"pm.info.iteration"',
+        _: '"require(\'lodash\')"',
+        CryptoJS: 'global "crypto" object',
```

Comment:
1. removed alternative for Backbone as the lib is itself deprecated now
2. Update alternative for CryptoJS